packages/frontend/editor-ui/src/features/ai/mcpAccess/components/header/connectPopover/MCPAccessTokenPopoverTab.vue

```diff
@@ -149,6 +150,9 @@ onMounted(async () => {
 					</N8nTooltip>
 				</template>
 			</ConnectionParameter>
+			<N8nNotice>
```

Comment:
I think we should show the notice only when key can be copied:
```suggestion
			<N8nNotice v-if="!isKeyRedacted">
```

But with this we don't know how to size the loading skeletons (it's not certain if the notice will be there after loading), so we'll have some layout shifts.
It's really a nitpick but it's quite nice how nothing moves in the current version.
Alternatively, we could just show warning icon next to the access token with a tooltip (similar to parameter input with issues)